CMSgov/design-system
# NO-TICKET Deprecate Sketch file in GitHub repo

## Summary

This RS is some cleanup of the design assets folder. Eventually that folder will go away in the repo. This first step is deprecation. 

We've been using Sketch Cloud to house our UI kits for some time now. This PR updates the UI kit in GitHub to let folks know that it has been deprecated and how they can get the up to date UI kit from Sketch Cloud.

This PR also removes, Bitter, Font Awesome and Open Sans fonts from the repo as those elements are already embedded into the UI kits and already referenced in our codebase. 

### Deprecated

Deprecated the UI kit found in the repo and referenced the more up to date UI kit in Sketch Cloud

### Removed

- Removed Bitter, Font Awesome and Open Sans fonts from the repo  

## How to test

- See that the fonts have been removed 
- Open the sketch file and find the deprecation message and reference to the more up to date UI kit
- See the README has been updated with how to get the more up to date UI kit
